docsite/rst/committer_guidelines.rst

```diff
@@ -0,0 +1,72 @@
+Committers Guidelines (for people with commit rights to Ansible on GitHub)
+``````````````````````````````````````````````````````````````````````````
+
+These are the guidelines for people with commit access to Ansible. Committers are essentially acting as members of the Ansible Core team, although not necessarily as an employee of Ansible and Red Hat. Please read the guidelines before you commit.
+
+These guidelines apply to everyone. At the same time, this ISN’T a process document. So just use good judgement. You’ve been given commit access because we trust your judgement.
+
+That said, use the trust wisely. 
+
+If you abuse the trust and break components and builds, or waste a lot of time asking people to review incomplete or untested pull requests, the trust level falls and you may be asked not to commit or you may lose access to do so.
```

Comment:
everyone can submit incomplete or untested pull requests .. not sure why apply this to committers only